extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -100,10 +102,13 @@ public function getPages() {
 
 		$dbh = wfGetDB( DB_SLAVE );
 		( new \WikiaSQL() )
-			->SELECT_ALL()
+			->SELECT( "page.*, comments_index.*, IF(pp.props is NULL, page.page_id, REPLACE(REPLACE(REPLACE(pp.props, ';', ''), 'i', ''), ':', '')) as idx" )
```

Comment:
These nested `REPLACE` calls are pretty hideous, but they're the easiest way I found to transform a value like this: `i:123;` into `123`.